pincer/objects/channel.py

```diff
@@ -34,6 +33,12 @@
 from .overwrite import Overwrite
 from ..utils import APIObject, APINullable, MISSING, Snowflake, Timestamp
 
+if TYPE_CHECKING:
+    from pincer import Client

```

Comment:
doesn't `from . import Client` work here? i've seen you used `pincer` instead of `.` at multiples places

Reply:
I'm assuming `from . import Client` will work unless it's out of TYPE_CHECKING but I have not tested that yet.